acceptancetests/assess_caas_bootstrap.py

```diff
@@ -40,14 +40,14 @@ def assess_caas_bootstrap(client):
         juju_ver=client.version
     )
 
-    caas_client = deploy_caas_stack(bundle_path=bundle, client=client, timeout=3600)
+    caas_client = deploy_caas_stack(path=bundle, client=client, timeout=3600)
```

Comment:
@ycliuhw sorry this is my bad. I do apologise.

Reply:
No worries! it's just a simple fix.